cli/src/commands/git/push.rs

```diff
@@ -79,6 +79,15 @@ use crate::ui::Ui;
 /// bookmarks. Use `--all` to push all bookmarks. Use `--change` to generate
 /// bookmark names based on the change IDs of specific commits.
 ///
+/// When pushing a bookmark, the command pushes all commits in the range from
+/// the remote's current position up to and including the bookmark's target
+/// commit. Any descendant commits beyond the bookmark are not pushed. For
+///
+/// If the local bookmark has diverged from the remote (non-fast-forward), the
```

Comment:
I think we should avoid saying "diverged" here because that suggests to me that both local and remove side have moved and here it's just the local side that has moved. Maybe "If the local bookmark has changed since the last fetch"? Does that capture what you meant to say?

Reply:
Yeah that would work well, I was trying to motivate why you might end up getting a force push.